archive/readme.md

```diff
@@ -38,7 +38,7 @@ Then add the cards into it.
 
 ## How does it work
 
-The `getArchiveCards()` method creates `<object>` elements. As many as there are archive files (based on the manually updated `NUMBER_OF_FILES`).  
+The `getArchiveCards()` method creates `<object>` elements as many as there are archive files (based on the manually updated `NUMBER_OF_FILES`).  
```

Comment:
Hello @rahulsust, I am not sure that this change makes this sentence any clearer. @Syknapse just so the intention is clear., Would writing this sentence as follows clear up and confusion?

The `getArchiveCards()` method creates an `<object>` element for each archived file. The total number of archived files is based on the manually updated `NUMBER_OF_FILES`.

Let me know if this is conveys your message :)

Reply:
Yeah, yours is definitely clearer and improves in the original 👍🏼

Reply:
@rahulsust Looks like this sentence structure is what Syk (project owner) would like to go with. If you would like to make the change and push a commit to this PR, I can get that merged. Otherwise let me know and I will make the code changes to the master branch.

This is what we would like it changed to:

The `getArchiveCards()` method creates an `<object>` element for each archived file. The total number of archived files is based on the manually updated `NUMBER_OF_FILES`.

Thanks again for all your help with the documentation. 💪